local/run.go

```diff
@@ -416,6 +416,10 @@ func (a *Agent) Run(ctx context.Context) error {
 	// Windows requires certain env variables to be present
 	if runtime.GOOS == "windows" {
 		cmd.Env = append(cmd.Env,
+			"APPDATA="+os.Getenv("APPDATA"),
+			"LOCALAPPDATA="+os.Getenv("LOCALAPPDATA"),
+			"HOMEPATH="+os.Getenv("HOMEPATH"),
+			"USERPROFILE="+os.Getenv("USERPROFILE"),
```

Comment:
I'm assuming this works okay if those `os.Getenv()` calls return `""` empty string — it seems we're already making that assumption with 8 vars below this, so should be fine 👍🏼